### ISSUE TITLE
Enhance and expand standard RB and its features

### ISSUE THREAD

**Issue by wilkensJ** [qiboteam/qibocal#374]
Since the standard randomized benchmarking protocol PR #349 is close to being ready to be merged to main, I would like to collect all the ideas/requests/feedback which came up during the review and go beyond the scope of this PR. Thanks to @Jacfomg and @andrea-pasquale for the feedback, if something is missing here please feel free to add points.

- [x] Make the global qubits statement in the qibocal runcard accessible to the data acquisition function. For that the `RBParameters` needs to be updated/`qubits` attribute overwritten when None is given for the qubits in the initialization in the acquisition function.
- [x] Make the nshots be readable from the platform when not specified in the runcard.
- [ ] Estimate SPAM error using A and B from the exponential fit.
- [ ] Add T1 and T2 to the table
- [x] Write a module using pre transpiled gates and an own compiler to have full control of the pulses. Maybe add unrolling of sequences as an option.

**Comment by andrea-pasquale** [comment]
@Jacfomg can we close this?

**Comment by Jacfomg** [comment]
I don't think we are estimating SPAM yet, for T1 and T2 table I feel the monitoring tool would make a better job that reporting it there. For the last, I am not aware of the compiler thing they wanted to make, but we do have unrolling.

**Comment by andrea-pasquale** [comment]
Closing this since it is currently outdated as an issue.
